watson/cli.py

```diff
@@ -1067,19 +1125,22 @@ def _final_print(lines):
             )
         )
 
-        _print("\n".join(
-            u"\t{id}  {start} to {stop}  {delta:>11}  {project}{tags}".format(
-                delta=format_timedelta(frame.stop - frame.start),
-                project=style('project', u'{:>{}}'.format(
-                    frame.project, longest_project
-                )),
-                tags=(" "*2 if frame.tags else "") + style('tags', frame.tags),
-                start=style('time', '{:HH:mm}'.format(frame.start)),
-                stop=style('time', '{:HH:mm}'.format(frame.stop)),
-                id=style('short_id', frame.id)
-            )
-            for frame in frames
-        ))
+        for frame in frames:
```

Comment:
I was not sure how to incorporate the if-statement in this section, so this is the only place in the merge where I went with the older syntax with the traditional for loop. Let me know if you prefer the have this change reverse and how it would be written with the 'note' if statement below.